tools/psi-visualizer/visualize.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Please, add license SPDX header and Copyright info....

Reply:
Always forgetting it. Will fix!

Reply:
Fixed